api-update/Jenkinsfile-test

```diff
@@ -40,15 +46,15 @@ pipeline {
                         script {
                             unstash "antivirus-api-update-jar"
                             sh "cp antivirus/target/scala-2.13/antivirus-api-update.jar /"
-                            def accountNumber = getAccountNumberFromStage()
+                            def accountNumber = tdr.getAccountNumberFromStage(params.STAGE)
                             // sh "python /upload_to_s3.py ${accountNumber} ${params.STAGE} tdr-backend-code-mgmt ${versionTag}/antivirus-api-update.jar /antivirus-api-update.jar"
                             sh "aws s3 cp /antivirus-api-update.jar s3://tdr-backend-code-mgmt/${versionTag}/antivirus-api-update.jar"
 
                             sh 'git config --global user.email tna-digital-archiving-jenkins@nationalarchives.gov.uk'
                             sh 'git config --global user.name tna-digital-archiving-jenkins'
-                            sh "git tag v${versionTag}"
+                            sh "git tag ${versionTag}"
```

Comment:
What was going wrong with the old tag format? It would be nice to stick with the old format if possible for consistency with the frontend and API projects.

Reply:
Oh yeah. Sorry, I missed that there was a duplicate "v".